### ISSUE TITLE
Can't figure out working with the advanced usage using `build.gradle.kts`

### ISSUE THREAD

**Issue by StylianosGakis** [usefulness/easylauncher-gradle-plugin#426]
The section [advanced usage](https://github.com/usefulness/easylauncher-gradle-plugin#advanced-usage) shows a syntax like this:
```groovy
easylauncher {
    buildTypes {
        beta {
            // Add two more filters to all `beta` variants
            filters = [
                    customRibbon(ribbonColor: "#0000FF"),
                    overlayFilter(file("example-custom/launcherOverlay/beta.png"))
            ]
        }
        canary {
            // Remove ALL filters to `canary` variants
            enable false
        }
        release {}
    }
}
```
However in the kts file, when I try to do:
```kotlin
easylauncher {
  buildTypes.findByName("staging") {
    setFilters.
  }
}
```
I do not get anything for `setFilters`, and I can't quite figure out what I am configuring wrong 🤔

**Comment by StylianosGakis** [comment]
Okay okay I am silly, getByName doesn't seem to have an Action as the second parameter to allow this trailing lambda syntax, but this works
```kotlin
buildTypes.findByName("staging")?.apply {
  setFilters(grayRibbonFilter("staging"))
}
```
I figured it out basically as soon as I typed all this down and went back to my project. Silly me, all I needed was a rubber duck instead of help 😅

**Comment by mateuszkwiecinski** [comment]
Yeah, something like that should work. Although, I'm not sure if the plugin _registers_ any objects within `easylauncher.buildTypes`. The groovy equivalent from readme calls `register("")` under the hood, so you may also need that in your case
```kotlin
buildTypes.register("staging") {
  setFilters(grayRibbonFilter("staging"))
}
```

Similar issue: https://github.com/usefulness/easylauncher-gradle-plugin/issues/57#issuecomment-757052135

**Comment by StylianosGakis** [comment]
Yep yep I just figured that I was yet again doing something wrong as soon as I saw that it was not working 😅
I confused buildTypes for the one registered inside the android.buildTypes block, and thought I could just "get" that one directly.
Thanks a ton for the quick help, adjusting it to `register` (I had `create` first, not sure if there's a difference), does work perfectly fine!
Also good to see that this *just works* with the monochromatic icons too. The labels aren't colored (duh) but it works! So convenient, thanks so much for this plugin!
